packages/app/.changeset/nasty-fans-remember.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/app': patch
+---
+
+added extensions.targeting.preloads
```

Comment:
you need to run `pnpm changeset add` from the root, if you run it inside `packages/app` it won't work and adds a few extra files. (You'd need to delete `packages/app/.changeset`)